frontend/src/libs/components/loader/loader.tsx

```diff
@@ -3,19 +3,31 @@ import { getValidClassNames } from '~/libs/helpers/helpers.js';
 import styles from './styles.module.scss';
 
 type Properties = {
-  children: React.ReactNode;
+  children?: React.ReactNode;
   className?: string;
   isLoading: boolean;
+  hasOverlay?: boolean;
+  loaderType: 'dots' | 'circular';
```

Comment:
Maybe it would be better to name this prop as just `type`, I guess it’s self explanatory as well when passing it to `Loader` component